avocado/core/jobdata.py

```diff
@@ -41,6 +43,20 @@
 CMDLINE_FILENAME = 'cmdline'
 
 
+def _find_class(module, name):
+    """
+    Look for a class including compatibility workarounds
+    """
+    if module == "avocado.plugins.yaml_to_mux":
+        mod = __import__("avocado_varianter_yaml_to_mux", fromlist=[module])
+        return getattr(mod, name)
+    else:
+        mod = __import__(module)
+        mod = sys.modules[module]
+        klass = getattr(mod, name)
+        return klass
```

Comment:
No need to have this `klass` variable, but that's minor. Wonder why did you remove the absolute-imports-only (`level=0`) restriction from the original `Unpickler.find_class()`. Any specific reason?